index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nelson Araujo's Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <header>
+        <section class="mobile-nav">
+            <div class="landing-header">
+                <div class="header-image"></div>
+                <button id='open-menu-btn' class="menu-btn">
+                    <img src="./assets/ic_menu.svg">
+                </button>
+            </div>
+            <nav id='menu-container'>
+                <div class="top-image"></div>
+                <button id='close-menu-btn' class="menu-btn">
+                    <img src="./assets/Union.svg">
+                </button>
+                <ul>
+                    <li>
+                        <a>Hello</a>
+                    </li>
+                    <li>
+                        <a>Portfolio</a>
+                    </li>
+                    <li>
+                        <a>About</a>
+                    </li>
+                    <li>
+                        <a>Contact</a>
+                    </li>
+                </ul>
+                <div class="bottom-image"></div>
+            </nav>
+        </section>
+    </header>
+    <main>
+        <section class="introduction-mobile">
+            <h1>Nelson Araujo</h1>
```

Comment:
According to the provided template, you were supposed to be using `Playfair display` as a choice `font-family` but you are using `Poppins`.
I recommend using the choices `font-family` to keep your UI consistent with the Figma design so your website UI will be impeccable